app/views/idv/shared/_ssn.html.erb

```diff
@@ -5,6 +5,7 @@ locals:
 * success_alert_enabled: whether or not to display a "We've successfully verified your ID" success alert
 * updating_ssn: true if the user is updating their SSN instead of providing it for the first time. This
                 will render a different page heading and different navigation buttons in the page footer
+* flow: Current flow; used to go to the previous state in the correct flow with "Back" link
```

Comment:
Where do we use this `flow` value?

Reply:
It was used to determine the current flow, but is no longer needed. Removed.